2-mandatory/1-recipes.js

```diff
@@ -20,6 +20,55 @@
   cocoa
 
   You should write and log at least 5 recipes
+   
+
 */
 
-// write code here
\ No newline at end of file
+let recipe = {
+  title :"Mole",
+  servings : 2 ,
+  ingredients : [
+    "cinnamon" , "cumin" , "cocoa"
+  ],
+  displayRecipe : function (){
+    console.log(this.title);
+    console.log("Serves : "+ this.servings);
+    console.log("Ingredients:");
+    for (item of this.ingredients){
+      console.log(item);
+    }
+  }
+}
+let recipe1 = {
+  title :"Biryani",
+  servings : 2 ,
+  ingredients : [
+    "rice" , "spices" , "meat"
+  ],
+  displayRecipe : function (){
+    console.log(this.title);
+    console.log("Serves : "+ this.servings);
+    console.log("Ingredients:");
+    for (item of this.ingredients){
+      console.log(item);
+    }
+  }
+}
+let recipe2 = {
+  title :"Anda Paratha",
+  servings : 2 ,
+  ingredients : [
+    "eggs" , "spices" , "atta" , "oil"
+  ],
+  displayRecipe : function (){
+    console.log(this.title);
+    console.log("Serves : "+ this.servings);
+    console.log("Ingredients:");
+    for (item of this.ingredients){
+      console.log(item);
+    }
+  }
+}
+recipe.displayRecipe();
+recipe1.displayRecipe();
+recipe2.displayRecipe();
```

Comment:
great job and a great idea, but the question did not want you to have a method inside of each recipe, it is important to do what the questions ask us to do, you should find a way to go through each recipe and console.log the recipe items.
hint: you can have array of objects